corehq/apps/accounting/invoicing.py

```diff
@@ -418,9 +418,12 @@ def base_description(self):
     @property
     def unit_description(self):
         if self.is_prorated:
-            return _("%(num_days)s day%(pluralize)s of %(plan_name)s Software Plan.") % {
+            return ungettext(
+                "%(num_days)s day of %(plan_name)s Software Plan.",
+                "%(num_days)s days of %(plan_name)s Software Plan.",
+                self.num_prorated_days
+            ) % {
```

Comment:
As an aside, do you have a strong preference for `"Hello %(name)s" % name` over `"Hello {name}".format(name)`?  I tend to prefer the latter, as I find it more readable (translators can get confused by that trailing `s`) and flexible, but curious if you have arguments against it.